test/token/ERC1155/ERC1155.test.js

```diff
@@ -9,12 +9,12 @@ const { shouldBehaveLikeERC1155 } = require('./ERC1155.behavior');
 const ERC1155Mock = contract.fromArtifact('ERC1155Mock');
 
 describe('ERC1155', function () {
```

Comment:
Since the changes in this file are mostly cosmetic, maybe you want to revert them so they won't conflict with #2107.